<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1288</width>
    <height>676</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonSelectInFolder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Select In Folder</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEditInDirectory">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>821</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSelectOutFolder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Out Folder</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEditOutDirectory">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>821</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="ListWidgetFiles">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>411</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="CheckBoxShowInputImageGray">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>181</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Show input image Gray</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="CheckBoxShowInputImagePC">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>100</y>
      <width>181</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Show input image PC</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxMinShowPseudoColor">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>190</y>
      <width>70</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="minimum">
     <number>-65536</number>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
    <property name="singleStep">
     <number>2</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxMinShowGray">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>130</y>
      <width>70</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="minimum">
     <number>-65536</number>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
    <property name="singleStep">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelMaxDispGray">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>160</y>
      <width>172</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Max disp Gray</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMaxDispPseudoColor">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>220</y>
      <width>172</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Max  display pseudo color</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMinDispGray">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>130</y>
      <width>172</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Min disp Gray&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxMaxShowPseudoColor">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>220</y>
      <width>70</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
    <property name="singleStep">
     <number>32</number>
    </property>
    <property name="value">
     <number>4095</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxMaxShowGray">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>160</y>
      <width>70</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
    <property name="singleStep">
     <number>32</number>
    </property>
    <property name="value">
     <number>4095</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelMinDispPseudoColor">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>190</y>
      <width>172</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Min display pseudo color&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxPreprocessType">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>250</y>
      <width>251</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxTileShape">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>310</y>
      <width>251</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxPreprocessKernelSize">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>280</y>
      <width>70</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="labelMaxPreprocessKernelSize">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>280</y>
      <width>172</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Preprocess Kernel size</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTileSize">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>340</y>
      <width>172</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Tile size&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOffsetY">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>430</y>
      <width>172</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Tile offset y&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOffsetX">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>400</y>
      <width>172</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Tile offset x&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelShift">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>370</y>
      <width>172</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Tile shift&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CheckBoxShowTiles">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>70</y>
      <width>181</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Tiles</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelTileLineWidth">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>100</y>
      <width>172</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Tile line width&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CheckBoxShowDirection">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>130</y>
      <width>181</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Direction</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelDirectionLineWidth">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>160</y>
      <width>172</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Direction line width&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMinOffset">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>290</y>
      <width>172</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Min offset&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOffsetCount">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>320</y>
      <width>172</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Offsets count&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOffsetStep">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>350</y>
      <width>172</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Offset step&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxNormalisation">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>220</y>
      <width>251</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelBinCount">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>260</y>
      <width>172</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Bin count&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDirectionLileLength">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>190</y>
      <width>172</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Direction line lenght&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxTileSize">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>340</y>
      <width>70</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>3</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxTileOffsetX">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>400</y>
      <width>70</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxTileOffsetY">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>430</y>
      <width>70</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxTileShift">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>370</y>
      <width>70</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxTileLineWidth">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>100</y>
      <width>70</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxDirectionLineWidth">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>160</y>
      <width>70</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxDirectionLineLenght">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>190</y>
      <width>70</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxBinCount">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>260</y>
      <width>70</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxMinOffset">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>290</y>
      <width>70</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxOffsetCount">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>320</y>
      <width>70</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxOffsetStep">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>350</y>
      <width>70</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="labelAngleStep">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>380</y>
      <width>172</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Angle step&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBoxAngleStep">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>380</y>
      <width>70</width>
      <height>24</height>
     </rect>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEditFilePattern">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>550</y>
      <width>281</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelFilePattern">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>172</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;File Pattern&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CheckBoxCalculateDirectionality">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>410</y>
      <width>181</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate Directionality</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="CheckBoxShowOutputText">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>430</y>
      <width>181</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Output</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="CheckBoxShowOutputTile">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>450</y>
      <width>181</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Output tile</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditOutput">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>480</y>
      <width>831</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1288</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
